### ISSUE TITLE
Do Not Print Amazon EMR Credentials to stderr

### ISSUE THREAD

**Issue by yiransheng** [Yelp/mrjob#1353]
Currently when running on EMR with `--verbose` flag, mrjob outputs AWS credentials to stderr. 

Example:

```
Active configuration:
{'additional_emr_info': None,
 'ami_version': '3.11.0',
 'aws_access_key_id': **********************,
 'aws_availability_zone': None,
 'aws_region': 'us-east-1',
 'aws_secret_access_key': ***************************,
 'aws_security_token': None,
 'bootstrap': [],
 'bootstrap_actions': [],
 'bootstrap_cmds': [],
 'bootstrap_files': [],
 'bootstrap_mrjob': None,
 'bootstrap_python': True,
 'bootstrap_python_packages': [],
 'bootstrap_scripts': [],
 'check_emr_status_every': 30,
 'check_input_paths': True,
```

I piped mrjob output to a file, and accidentally committed it to github... 

(things did not go pretty)

To prevent this happening, we should not print credentials not even with `--verbose` flag.

**Comment by davidmarin** [comment]
You're the first person to mention having that problem, but yeah, that makes sense.

**Comment by davidmarin** [comment]
What would you think of leaving the last 4 characters of the credentials, something like they do with credit cards? That way, users could easily tell which of their credentials are being used, without other people who see the logs being able to use them.

**Comment by davidmarin** [comment]
Adding to next release; this should be easy.

**Comment by yiransheng** [comment]
Good idea on the first 4 characters solution.

Thanks for taking a look!

**Comment by davidmarin** [comment]
This is done, available in the `master` branch, and will be in v0.5.4.

**Comment by davidmarin** [comment]
This is in v0.5.4.
